README.md

```diff
@@ -30,6 +30,17 @@ pip install -r requirements.txt
 
 This will install all dependencies required to run the python server.
 
+For Deepspeech transcription to work, you will need to also add the pre-trained model and scorer to `src/py_server`
```

Comment:
Since there was a pull request that refactored the code and directories to follow a camelCase naming convension could you change `src/py_server` to `src/pyServer`